test/library/randomized_benchmarking/test_rb_analysis.py

```diff
@@ -70,8 +71,7 @@ def setUp(self):
             backend=backend,
         )
         exp_1qrb_q0.set_transpile_options(**transpiler_options)
-        expdata_1qrb_q0 = exp_1qrb_q0.run(analysis=None)
-        self.assertExperimentDone(expdata_1qrb_q0, timeout=300)
+        expdata_1qrb_q0 = exp_1qrb_q0.run(analysis=None).block_for_results()
```

Comment:
Hmm, what's the reason for changing `assertExperimentDone` to `block_for_results`? `assertExperimentDone` calls `block_for_results` and also checks that all threads completed successfully.

Reply:
Oh never mind, I see the logic, my bad.